DEV-INTRO.md

```diff
@@ -1,59 +1,166 @@
 
-### Virtual Machine
+#### .env file
+You can copy the [`env.example`](docker/env.example) file as .env in your pecan folder. The variables we want to modify are:
 
-* Fastest way to get started
-* see PEcAn demo ...
+* `COMPOSE_PROJECT_NAME` set this to pecan, the prefix for all containers
+* `PECAN_VERSION` set this to develop, the docker image we start with
 
-### Installing from source
+#### folders
 
-#### From GitHub
+Next we will create the folders that will hold all the data for the docker containers using: `mkdir -p volumes/{lib,pecan,portainer,postgres,rabbitmq,traefik}`. The `volumes` folder will be ignored by git. You can create these at any location, however you will need to update the `docker-compose.dev.yml` file. The subfolders are used for the following:
+
+- **lib** holds all the R packages for the specific version of PEcAn and R. This folder will be shared amongst all other containers, and will contain the compiled PEcAn code.
+- **pecan** this holds all the data, such as workflows and any downloaded data.
+- **portainer** if you enabled the portainer service this folder is used to hold persistent data for this service
+- **postgres** holds the actual database data. If you want to backup the database, you can stop the postgres container, zip up the folder.
+- **rabbitmq** holds persistent information of the message broker (rabbitmq). 
+- **traefik** holds persisent data for the web proxy, that directs incoming traffic to the correct container.
+
+These folders will hold all the persistent data for each of the respective containers and can grow. For example the postgres database is multiple GB. The pecan folder will hold all data produced by the workflows, including any downloaded data, and can grow to many giga bytes.
+
+#### postgresql database
+
+First we bring up postgresql (we will start RabbitMQ as well since it takes some time to start): `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d postgres rabbitmq`. This will start postgresql and rabbitmq. We need to wait for a few minutes (you can look at the logs using `docker-compose logs postgres`) to see if it is ready.
+
+Once the database has finished starting up we will initialize the database using: `docker run --rm --network pecan_pecan pecan/db`. Once that is done we create two users for BETY:
 
 ```
-library(devtools)
-install_github("pecan", "PEcAnProject")
+# guest user
+docker-compose run --rm bety user guestuser guestuser "Guest User" guestuser@example.com 4 4
+
+# example user
+docker-compose run --rm bety user carya illinois "Carya Demo User" carya@example.com 1 1
 ```
 
-#### "Makefile"
+#### copy web config file
+
+The `docker-compose.dev.yaml` file has a section that will eanble editing the web application. This is by default commented out. If you want to uncoment it you will need to first copy the config.php from the docker/web folder. You can do this using `cp docker/web/config.docker.php web/config.php`.
```

Comment:
not very clear if they must do this, or if this is just a suggestion..I think we won't use a stronger language we also need to explain what happens if they do/ if they don't